File: wc-at.py
# WC
import pandas as pd
# Importing the data 
wc = pd.read_csv("C:\\Users\\WIN10\\Desktop\\LEARNING\\DS\\ProbCLT\\probability dataset\\wc-at.csv")

#Graphical Representation
import matplotlib.pyplot as plt # mostly used for visualization purposes 
import numpy as np

plt.hist(wc.Waist) #histogram
plt.hist(wc.AT) #histogram

plt.boxplot(wc.Waist) #boxplot
plt.boxplot(wc.AT) #boxplot
plt.bar(height = wc.Waist, x = np.arange(len(wc)))# initializing the parameter
plt.bar(height = wc.AT, x = np.arange(len(wc)))# initializing the parameter

#Normal Quantile-Quantile Plot
import scipy.stats as stats
import pylab
# Checking Whether data is normally distributed
stats.probplot(wc.Waist, dist="norm",plot=pylab)
stats.probplot(wc.AT, dist="norm",plot=pylab)

File: wc-at.R
# wc
library(readr)
# Importing the dataset
wc <- read_csv("C:\\Users\\WIN10\\Desktop\\LEARNING\\DS\\ProbCLT\\probability dataset\\wc-at.csv")
mean(wc$Waist) 
mean(wc$AT)

# Probability Distribution
library(UsingR)
densityplot(wc$Waist)
densityplot(wc$AT)

# Normal Quantile-Quantile Plot
qqnorm(wc$Waist)
qqline(wc$Waist)
qqnorm(wc$AT)
qqline(wc$AT)

library(moments)
# Graphical Representation
barplot(wc$Waist)
dotchart(wc$Waist)
barplot(wc$AT)
dotchart(wc$AT)
# Histogram
hist(wc$Waist) 
hist(wc$AT) 
# Boxplot
boxplot(wc$Waist) 
boxplot(wc$AT) 


